src/error/MatterError.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { StatusCode } from "../matter/interaction/InteractionMessages";
+
+/** Error base class for all errors thrown by this library. */
+export class MatterError extends Error {}
+
+/** Specific Error for when a fabric is not found. */
+export class FabricNotFoundError extends MatterError {}
+
+
+/** Error base Class for all errors related to the status response messages. */
```

Comment:
This is specific to InteractionProtocol so should probably be in matter/interaction
Maybe part of the Messenger?

Reply:
Moved to a new FIle in interaction dir because I expect that it will be more later